brightlayer-ui/doc-it
# PXBLUE-2672: Remove "PX Blue" from Themes

Fixes PXBLUE-2672


#### Changes proposed in this Pull Request:
- Remove all PX Blue from themes; replace with BLUI


#### Screenshots:

Spring Festival:
![image](https://user-images.githubusercontent.com/8997218/149177069-15e50228-4f67-4cad-9bf1-4bb045d5e749.png)

Earth Day:
![image](https://user-images.githubusercontent.com/8997218/149176859-0b654ac3-309d-44ac-b83f-e3a11d851dab.png)

Mid-Autumn Festival
![image](https://user-images.githubusercontent.com/8997218/149176570-bcd44316-03f8-4d56-a088-ba4af2974989.png)


#### To Test:
- `yarn start`
- Change dates in the following, and observe the landing page change:
  - `src/__configuration__/themes/spring-festival.ts`, 
  - `src/__configuration__/themes/earth-day.ts`
  - `src/__configuration__/themes/mid-autumn-festival.ts`
